src/Core/tests/DeviceTests/Handlers/HandlerTestBase.cs

```diff
@@ -2,23 +2,48 @@
 using System.Threading.Tasks;
 using Microsoft.Maui.DeviceTests.Stubs;
 using Microsoft.Maui.Essentials;
+using Microsoft.Maui.Hosting;
 using Xunit;
 
 namespace Microsoft.Maui.DeviceTests
 {
-	[Collection(TestCollections.Handlers)]
-	public partial class HandlerTestBase<THandler, TStub> : TestBase
+	public partial class HandlerTestBase<THandler, TStub> : TestBase, IDisposable
 		where THandler : IViewHandler
 		where TStub : StubBase, IView, new()
 	{
-		readonly HandlerTestFixture _fixture;
+		AppStub _app;
+		IAppHost _host;
+		IMauiContext _context;
 
-		public HandlerTestBase(HandlerTestFixture fixture)
+		public HandlerTestBase()
```

Comment:
Because there is no need for a static App instance, we no longer need to care that we are testing multiple apps at the same time because all the magic is now in the builder and the base `App` type is pure DI capable. ❤️